tests/test_env_envp.c

```diff
@@ -6,14 +6,14 @@
 /*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/06/28 08:17:46 by gwolf             #+#    #+#             */
-/*   Updated: 2023/07/01 18:28:57 by gwolf            ###   ########.fr       */
+/*   Updated: 2023/07/14 18:36:37 by gwolf            ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "test.h"
 
-t_hashtable	*symtab;
-char		**envp;
+extern t_hashtable	*symtab;
+extern char			**envp;
 
 void	test_env_envp_two(void)
```

Comment:
What is the functionality of these two functions?

Reply:
With the extern keyword i can "import" the globals from main. The main has some vars globally defined, which can be used in the testfile. I like to use it for t_hashtable *symtab for example. Then I don't need to pass the pointer around, I can just use it.

Reply:
I was referring to the functions:

void	test_env_envp_two(void)
void	test_env_envp_empty(void)

I guess these are to create the environment, one with two variables and one which is empty

Reply:
Yeah these are testcases for creation of envp: two vars in environment and empty environment.